### DISCUSSION TITLE
Can't Access Management Portal

### DISCUSSION THREAD

**Discussion by bbaluyut** [postalserver/postal#1974] [Help with using Postal]
Server is Ubuntu hosted on prem with an ip address of 192.168.0.10. Using Caddy.
I can access the portal using 127.0.0.1:5000 with in Ubuntu.
However if I use another device in the same subnet and type http://192.168.0.10:5000, I can't.
I can ping it via IP and hostname but can't access via http/https?
 
Is this not possible. Overall I'm trying to setup a local smtp server.

**Comment by willpower232** [comment]
this is how ports work, it is bound to the local server by default and cannot be accessed outside unless it is attached to `0.0.0.0` or you use a proxy such as caddy or nginx to forward the port outside of the server.

**Reply by bbaluyut** [reply]
so what options do i have to access the portal without an external DNS provider?

**Reply by willpower232** [reply]
can you not install nginx or caddy?

**Reply by bbaluyut** [reply]
so if skip the caddy part in the instructions i can access the portal via IP locally?
